internal/core/core_fence.go

```diff
@@ -0,0 +1,79 @@
+package core
+
+/*
+#cgo LDFLAGS: -lopenvds
+#cgo CXXFLAGS: -std=c++17
+#include <capi.h>
+#include <ctypes.h>
+#include <stdlib.h>
```

Comment:
Didn't think that we would have to include all these in every file :pouting_cat: 

Wondering what is better - leave it as is or always include only capi.h, as it in itself actually has import of ctypes and stdlib. Probably would depend on compiler though if it compiles or not.

So I would probably vote for what you currently have, but would gladly hear other opinions :)